### PR TITLE
Update CONTRIBUTING.md to state targeting live for localized repos

### LINKED ISSUE #9770: CONTRIBUTING.md is written for contribution toward English repo only
**Issue description**

[CONTRIBUTING.md](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) is written for contribution toward English repo only, so when it was localized, it caused confusion for contributor toward localized repo. 
Here is the GitHub Issue raised for reference: [https://github.com/dotnet/docs.ja-jp/issues/275](https://github.com/dotnet/docs.ja-jp/issues/275)  

The sentence that caused confusion is, "**Step 4:** Submit a Pull Request (PR) from your branch to `dotnet/docs/master`." and "Your PR should *always* target the master branch. You should *never* open a PR that targets the live branch."  

For contribution toward localized languages such as Japanese, PR should target the `live` branch where the `Edit` button is set to take them. GitHub repos for localized languages do not use `master` branch and only use `live` branch, so contribution is setup to open PR in `live` branch.

Can you please help to add this exception to the contributor guide?

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #9770

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ docs
 
 Your PR should *always* target the master branch. You should *never* open a PR that targets the live branch.
```
Comment (mairaw) [inline CONTRIBUTING.md]: @BillWagner how would you feel about a rewrite like this instead?

```suggestion
Your PR should *always* target the default branch of the repository (unless you're working on a release branch). For dotnet/docs, master branch is the default branch. For the localized repositories, live branch is the default one. You should *never* open a PR that targets the live branch on dotnet/docs.
```

Comment (Youssef1313) [reply CONTRIBUTING.md]: @mairaw, This rewrite LGTM, waiting @BillWagner feedback before applying it.

Comment (BillWagner) [reply CONTRIBUTING.md]: I agree with @mairaw 's suggestion.

**Review by BillWagner — APPROVED** [review]
This LGTM.

I'll :shipit: now.

**Pull request merged**